### PR TITLE
Feature/input prefix

### PR DESCRIPTION
**Pull request by kevinch**
screen shot 2018-05-17 at 13 23 21

### CODE REVIEW

**Comment by kevinch** [comment]
HOLD. @cmdalbem & @rsimoens working on it.

**Comment by rsimoens** [comment]
Demos uma estudada aqui, vamos mudar um pouco esse visual @kevinch:

image

Layout:
https://www.figma.com/file/a94lX91ZmYGIDBxvb8shwLn8/VTEX-Design?node-id=0%3A1

**Comment by kevinch** [comment]
screen shot 2018-05-28 at 17 44 22

File: src/components/icon/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ const Warning = require('./Warning').default;
 <div className="flex flex-wrap tl">
   <div class="mr6 mb6">
```
Comment (leticiawanderley) [inline src/components/icon/README.md]: Use `className`?

**Pull request merged**